README.rst

```diff
@@ -4,15 +4,15 @@ GenPy: An AST for Python
 .. image:: https://gitlab.tiker.net/inducer/genpy/badges/main/pipeline.svg
     :alt: Gitlab Build Status
     :target: https://gitlab.tiker.net/inducer/genpy/commits/main
-.. image:: https://github.com/inducer/genpy/workflows/CI/badge.svg?branch=main&event=push
+.. image:: https://github.com/inducer/genpy/actions/workflows/ci.yml/badge.svg
     :alt: Github Build Status
-    :target: https://github.com/inducer/genpy/actions?query=branch%3Amain+workflow%3ACI+event%3Apush
-.. image:: https://badge.fury.io/py/genpy.png
+    :target: https://github.com/inducer/genpy/actions/workflows/ci.yml
+.. image:: https://badge.fury.io/py/genpy.svg
     :alt: Python Package Index Release Page
     :target: https://pypi.org/project/genpy/
-.. image:: https://zenodo.org/badge/58979871.svg
+.. image:: https://zenodo.org/badge/DOI/10.5281/zenodo.6991526.svg
     :alt: Zenodo DOI for latest release
-    :target: https://zenodo.org/badge/latestdoi/58979871
+    :target: https://doi.org/10.5281/zenodo.6991526
```

Comment:
Updated the DOI to point to the latest one the way that Zenodo is recommending it now: https://zenodo.org/records/6991527.